app/talk/comment-image-selector.cjsx

```
@@ -19,6 +20,7 @@ module?.exports = React.createClass
       .then (subjects) =>
         subject.type = 'recent' for subject in subjects
         @setState {subjects}
+      .catch =>
```

Were these empty catch functions added for anything specific?